### ISSUE TITLE
Make it Responsive

### ISSUE THREAD

**Issue by Darshanjasani73** [Darshanjasani73/SpeakEasy2.0#1]
The SpeakEasy 2.0 application currently lacks full responsiveness, leading to suboptimal user experience on different devices, especially mobile and tablet screens. To enhance UI/UX, user accessibility and usability, it is crucial to implement responsive design principles. This feature request aims to make the application fully responsive across various screen sizes and improvement in UX.
